api/api/docs/base_docs.py

```diff
@@ -83,3 +98,123 @@ def get_operation_id(self) -> str:
             else:
                 operation_tokens.append("detail")
         return "_".join(operation_tokens)
+
+
+source_404_message = "Invalid source 'name'. Valid sources are ..."
+source_404_response = OpenApiResponse(
+    NotFoundErrorSerializer,
+    examples=[
+        OpenApiExample(
+            name="404",
+            value={"detail": source_404_message},
+        )
+    ],
+)
+
+
+def build_source_path_parameter(media_type: MediaType):
+    valid_description = (
+        f"Valid values are source_names from the stats endpoint: "
+        f"https://api.openverse.engineering/v1/{media_type}/stats/."
+    )
+
+    return OpenApiParameter(
+        name="source",
+        type=str,
+        location=OpenApiParameter.PATH,
+        description=f"The source of {media_type}. {valid_description}",
+    )
+
+
+creator_path_parameter = OpenApiParameter(
+    name="creator",
+    type=str,
+    location=OpenApiParameter.PATH,
+    description="The name of the media creator. This parameter "
+    "is case-sensitive, and matches exactly.",
+)
+tag_path_parameter = OpenApiParameter(
+    name="tag",
+    type=str,
+    location=OpenApiParameter.PATH,
+    description="The tag of the media. Not case-sensitive.",
+)
+
+
+def get_collection_description(media_type, collection):
+    if collection == "tag":
+        return f"""
+Get a collection of {media_type} with a specific tag.
+
+This endpoint returns only the exact matches. To search within the
+tag values, or to match several tags, use the `search` endpoint
+with `tags` query parameter instead of `q` parameter.
+
+The returned results are ordered primarily based on their popularity
+and authority. However, note that the exact order may vary over time
+or across requests.
```

Comment:
I was trying to add a statement to show that the order can change and is not constant. Since the popularity and authority boosts were removed from tags, I updated the docs to only include the order added to Openverse.